sequencer/src/options.rs

```diff
@@ -104,6 +104,32 @@ pub struct Options {
     #[derivative(Debug(format_with = "Display::fmt"))]
     pub state_relay_server_url: Url,
 
+    /// URL of the Auction Results Solver
+    #[clap(
+        long,
+        env = "ESPRESSO_AUCTION_RESULTS_SOLVER_URL",
+        default_value = "http://localhost:8083"
+    )]
+    #[derivative(Debug(format_with = "Display::fmt"))]
+    pub auction_results_solver_url: Url,
+
+    /// URL of generic builder
+    #[clap(
+        long,
+        env = "ESPRESSO_GENERIC_BUILDER_URL",
```

Comment:
We have renamed generic builder to fallback builder

Reply:
Since this is used for `MarketplaceConfig` I would prefer to wait until the name gets changed there. Which I don't believe has happened yet.